### ISSUE TITLE
sh: syntax error: EOF in backquote substitution

### ISSUE THREAD

**Issue by pl77** [nZEDb/nZEDb#2604]
FYI: I just installed the dev branch, so I'm not certain if this is a recent issue as I do not recall seeing it when I was running 0.8.12.0.  Take this report as merely feedback for the current dev branch, if nothing else.  Otherwise everything seems to be working fine. 

###Expected behaviour
`/misc/update/nix/multiprocessing/php postprocess.php add` should process all releases for additional information. 

###Actual behaviour
Majority of processing is happening with no issue, but periodically I receive the following output:
```sh
[472357][27.1MB]
[533602][29.3MB](cB)r(cB)r(cB)r(cB)r(cB)r(cB)r(cB)r(cB)r(cB)r(cB)r(cB)r(cB)r(cB)r(cB)r(cB)r(cB)r(cB)r(cB)rsh: syntax error: EOF in backquote substitution
sh: syntax error: EOF in backquote substitution
(cB)r(cB)r(cB)r(cB)r(cB)r(cB)r
[324548][202.5MB]
[594007][52MB]
```
###Steps to reproduce the behaviour
I am struggling to reproduce the error as it's so transient and unpredictable. I have attempted to backtrack through the code to see where postprocess might be calling with an unescaped backquote but not having much luck.

**Comment by niel** [comment]
Can you enable debugging (.../configuration/settings.php, copied from settings.example.php if you haven't already), and see if that provides more info on where this is happening.

**Comment by pl77** [comment]
Will do.  I will submit a comment as soon as I get some output.

Edit: debug showed a bunch of other errors (related to inadequate sphinx installation, nothing to do with nzedb) but so far haven't caught this one.

**Comment by pl77** [comment]
The only error I'm managing to find is in php_errors_cli.log which states:
```
PHP Notice:  Undefined offset: 1 in /var/www/nZEDb/libraries/zeebinz/rarinfo/rarinfo.php on line 1030
```
Even with all logging and debugging set I only get `sh: syntax error: EOF in backquote substitution` as an error message.

**Comment by pl77** [comment]
I have a feeling I just found the issue and it was unrelated to nZEDb.  Looking through the source file for zeebinz/rarinfo it seemed to be related to an old hack in the docker container I was using.  I've updated the container and so far so good.  I will leave this issue open for another day or so in case it returns or shows up in my logs.  

Regardless, thanks for the suggestion.  I wouldn't have found that issue if I didn't turn debug on and read the logs.
